commands/public/coinflip.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+	name: 'coinflip',
+	description: `Game: Coinflip. Which returns either heads or tails`,
+	execute: async function (client, message, args)
+	{
+
+		let coinArray = [
+			'Heads',
+			'Tails',
+		];
+
+		let answer = Math.floor(Math.random() * coinArray.length);
+
+		await message.channel.send(
```

Comment:
Use the message.channel.send as the following: 

```suggestion
		await message.channel.send({
		         embed: {
		               // etc
		         }
		});
```

Also, make sure the indenting is correctly